### PR TITLE
Fixes #13530: Full path details to shared-folders in WebUI

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/13530

### CODE REVIEW

File: techniques/fileDistribution/copyGitFile/2.0/metadata.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <SECTION name="Copy file" multivalued="false" component="true" componentKey="COPYFILE_NAME">
       <INPUT>
         <NAME>COPYFILE_NAME</NAME>
-        <DESCRIPTION>Path of the file to be copied</DESCRIPTION>
+        <DESCRIPTION>Path, relative to /var/rudder/configuration-repository/shared-files/</DESCRIPTION>
```
Comment (amousset) [inline techniques/fileDistribution/copyGitFile/2.0/metadata.xml]: I'd prefer to keep "Path to be copied" for clarity

Comment (amousset) [reply techniques/fileDistribution/copyGitFile/2.0/metadata.xml]: "source path" may be better, @peckpeck?

**Comment by VinceMacBuche** [comment]
I meant @Fdall ... sorry about the notification @felix ...

**Comment by Fdall** [comment]
Alexis review was already taken into account, the new github interface does not close the outdated reviews but only tag them with a small "outdated" in their right top corner...

**Comment by VinceMacBuche** [comment]
Damned ...

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**